lib/src/conversation.dart

```diff
@@ -149,18 +167,28 @@ class Conversation extends _BaseConversation {
             photoUrl: other.photoUrl,
             subject: other.subject);
 
-/* TODO: conversation.sendMessage is to be rewritten so that it works when we don't show the WebView
+  void _createConversation() {
+    if (!_conversationCreated) {
+      _session.execute('conversations["${id}"] = session.getOrCreateConversation("${id}")');
+
+      _conversationCreated = true;
+    }
+  }
+
   /// Sends a text message in a given conversation.
-  void sendMessage(String text, {Map<String, String>? custom}) {
-    final result = <String, dynamic>{};
+  Future<void> sendMessage(String text, {SendMessageOptions? options}) {
+    _createConversation();
```

Comment:
Let's remove this. It introduces an unnecessary side effect. The JSSDK already handles the scenario where the conversation id doesn't exist. We should just error out and propagate the JSSDK error from inside the webview.